### ISSUE TITLE
Blighted maps evaluation not working

### ISSUE THREAD

**Issue by ishinome** [Kyusung4698/PoE-Overlay#92]
For example on this map it shows nothing. It seems to put the name as blight and item category as skill gem

![image](https://user-images.githubusercontent.com/49497278/72474184-ec0a5400-37f0-11ea-96d8-e2f51e64c33c.png)

![image](https://user-images.githubusercontent.com/49497278/72474351-4acfcd80-37f1-11ea-9dd4-0d43632faa23.png)

```
Rarity: Rare
Forbidden Remnant
Blighted Port Map
--------
Map Tier: 6
Item Quantity: +72% (augmented)
Item Rarity: +31% (augmented)
Monster Pack Size: +35% (augmented)
Quality: +20% (augmented)
--------
Item Level: 76
--------
6 Blight Chests are Lucky
--------
Area is infested with Fungal Growths (implicit)
Natural inhabitants of this area have been removed (implicit)
--------
Area is inhabited by 2 additional Rogue Exiles
Monsters have 70% increased Area of Effect
Monsters Poison on Hit
Players have 25% less Armour
Players have 30% reduced Chance to Block
--------
Travel to this Map by using it in a personal Map Device. Maps can only be used once.
```

**Comment by Kyusung4698** [comment]
Thanks for reporting the issue 👍 This should be fixed with 0.5.1. Please report back!

**Comment by Kyusung4698** [comment]
I'll close this for now. Feel free to re-open if it's still not working.
